doc/CustomDataContainer.md

```diff
@@ -0,0 +1,64 @@
+# CustomDataContainer
+
+
+`CustomDataContainer` renders a plug'n'play custom data card.
+
+
+## Setup
+
+Before using this component in your project, `react-i18next`, `reactstrap` and `react-hook-form` must be installed and added into the project's `package.json` file:
```

Comment:
@petrKavulok I think here it is not necessary to mention it, since it can be used only within the seacat-admin-ui or where seacat-admin-ui is used as a submodule, but in all cases, you will need to have those libraries installed by default (it is already in package.json). Other case would be if it is a reusable component of asab-webui, but it is not.